### DISCUSSION TITLE
Node version

### DISCUSSION THREAD

**Discussion by kathirsvn** [stargate/stargate-mongoose#48] [General]
@vkarpov15 
With the latest [`mongodb` dependency ](https://github.com/stargate/stargate-mongoose/blob/cbd25c0949616520920e1e1947181d8ab5b9fa90/package.json#L91) to the code, it looks like the minimum version of node required is `Node 14` which has features like Optional chaining and nullish coalescing introducted.
https://www.ioannispoulakas.com/2020/10/18/node-14-optional-chaining-and-nullish-coalescing/.

So, in the README (especially in the Quick Start Section), should we explicitly mention that '14' (Node.js 14.x Fermium) is the minimum required version of NodeJS for using this library?

cc: @jeffreyscarpenter @amorton

**Comment by vkarpov15** [comment]
@kathirsvn setting Node 14 as the minimum required version is a good idea. You should also add an "engines" section to `package.json` so `npm` complains if you try to install stargate-mongoose when using Node 12.

**Reply by kathirsvn** [reply]
Thanks @vkarpov15. Sure.
